mantidproject/errorreports
# Slack filter to ignore any empty errorreports

Only if an errorreport has a non-default entry (such as Not provided) for the stacktrace, the textbox or the email will a notiication be sent to slack. With none of these we cannot resolve the problem.

To test:

On a local server (with mantid.properties: `errorreports.rooturl = http://localhost:8082`) and the .env file has the Slack line un-commented. Make sure to use Mantid workbench for producing errorreporter as only it can send a non-empty stacktrace.
check:
- No notification sent to Slack if Email, text and stacktrace are all "Not provided". Test this by loading `TrainingCourseData/Training_Exercise3a_SNS`, executing `NormaliseToMonitor` and clicking "Share Non-identifiable Info"
- Yes a notification is sent to Slack if Email and text are "Not provided", but Stacktrace is full. Test this by loading `TrainingCourseData/Training_Exercise3a_SNS`, executing `NormaliseToMonitor` and clicking "Yes, Share Info"
- Yes a notification is sent to Slack if Email and Stacktrace are "Not provided", but text is full. Test this by executing `Segfault`, setting text = Test and clicking "Yes, Share Info"
- Yes a notification is sent to Slack if Text and Stacktrace are "Not provided", but email is full. Test this by executing `Segfault`, setting email = Test and clicking "Yes, Share Info"
